Dockerfile-test

```diff
@@ -0,0 +1,3 @@
+FROM fxa-auth-server:build
+ENV FXA_L10N_SKIP 1
```

Comment:
This is already downloaded in the `Dockerfile-build` `npm install` step, but by the time we get here, we no longer have permission to remove files from `node_modules`, and the `download_l10n.sh` script causes npm to unconditionally remove a directory to try to replace it.

So, I settled on skipping it here.